jaeger/cmd/install_test.go

```diff
@@ -13,7 +13,7 @@ func TestRender(t *testing.T) {
 	// pin values that are changed by render functions on each test run
 	defaultValues := map[string]interface{}{
 		"webhook": map[string]interface{}{
-			"keyPEM":   "test-webhhook-key-pem",
+			"keyPEM":   "test-webhook-key-pem-ignored",
```

Comment:
Something odd happened here

Reply:
Oh, I meant to flag this one. The naive fix causes the tests to fail.

Reply:
*nearly* the same value.